runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheClientFetchFactory.java

```diff
@@ -887,16 +888,16 @@ private void doClientFanoutReplyWindow(
         private final long replyId;
         private final long leaderId;
         private final long authorization;
-        private final KafkaCacheCursor cursor;
         private final KafkaIsolation isolation;
         private final KafkaDeltaType deltaType;
         private final KafkaOffsetType maximumOffset;
         private final LongSupplier isolatedOffset;
         private final LongSupplier initialGroupIsolatedOffset;
 
+        private KafkaCacheCursor currentCursor;
+        private KafkaCacheCursor newCursor;
         private int state;
         private int flushOrDataFramesSent;
-
```

Comment:
Please revert blank line remove, it is intentional for readability to separate debitor fields.